### ISSUE TITLE
Always Timeout on iPod and iPhone 6

### ISSUE THREAD

**Issue by misuqian** [malcommac/SwiftLocation#28]
currentLocation always timeout on iPod and iPhone 6,and app will crash when timeout
here my code:
`  func updateLocation(errorHandler : (ErrorType -> Void)?){

```
    SwiftLocation.shared.cancelAllRequests()
    do{
        try SwiftLocation.shared.currentLocation(.Room, timeout: 10.0, onSuccess: {
            if let location = $0{
                self.reverseLocation(location)
            }
            }, onFail: { error in
                self.getSignificantLocation(errorHandler)
        })
    }catch(let error){
        print("Error \(error)")
    }
```

}`

**Comment by benjamincho** [comment]
It does not know exactly the project schedule.

First, "https://ip-api.com/json" is a problem. 

The need to modify "https" to "http".

Perhaps the problem is resolved.

But the problem can not be solved.

My problem was the error range of the location.

Check isAcceptable(_:) function

**Comment by malcommac** [comment]
Fixed in 0.9.0
